hconfig_utils/tests/Test_mapl3g_hconfig_get_private.pf

```diff
@@ -0,0 +1,225 @@
+module Test_mapl3g_hconfig_get_private
```

Comment:
No need to repeat the namespace protection here.
```suggestion
module Test_hconfig_get_private
```

Reply:
Done